### ISSUE TITLE
Add BatchNormalization operator for the Gorgonia Backend

### ISSUE THREAD

**Issue by owulveryck** [owulveryck/onnx-go#41]
**Is your feature request related to a problem? Please describe.**

Batch Normalization is a very common operator when using a neural network for image classification.
Most of the pre-trained model is using it.

The implementation has been initiated in the branch [batchnorm](https://github.com/owulveryck/onnx-go/tree/batchnorm)

**Describe the solution you'd like**
The `BacthNorm` should be able to run the tests described in the ONNX test backend:

```
go test -run=ONNX/TestBatchnorm
```

*Warning*, batchnorm's _scale_ and _biais_  may not have the same shape as the input; therefore, the operator should use "transparent" broadcasting.

**Describe alternatives you've considered**
N/A

**Additional context**

* [The ONNX doc for BatchNormalization](https://github.com/onnx/onnx/blob/master/docs/Operators.md#BatchNormalization)
* [The Gorgonia current implementation for BatchNorm](https://godoc.org/gorgonia.org/gorgonia#BatchNorm)
* [The description of the Batch Normalization described in the paper https://arxiv.org/abs/1502.03167](https://arxiv.org/abs/1502.03167)

**Comment by owulveryck** [comment]
Forward pass is implemented; closing this PR.
